priv/ie_gen_v1.erl

```diff
@@ -4,6 +4,7 @@
 
 -mode(compile).
 
+%Please check 8.1.1 chapter in "3GPP TS 29.244 rel 16"
```

Comment:
There is no written or syntax rule about comments, but the usual semantics for comments in Erlang code (derived from OTP code) seem to be:
* single % is indented by a lot (60 characters or so)
* double %% is indented to match the code indention
* triple %%% always start at the beginning of the line, no indentation at all
* there is always a space between the comment marker and text

Reply:
fixed